tests/integration/test_aws/__init__.py

```diff
@@ -0,0 +1,4 @@
+ONLY_LOGS_AFTER_PARAM = '--only_logs_after'
+
+
+local_internal_options = {'wazuh_modules.debug': '2', 'monitord.rotate_log': '0'}
```

Comment:
A test suite should not have `__init__.py`, these variables would go in the `__init__.py` of `aws`.